WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -28,15 +28,13 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
 
     @Published private(set) var orderStage: OrderStage = .building
 
-    private let currencyFormatter: CurrencyFormatter
-
     private let cardPresentPaymentService: CardPresentPaymentFacade
 
+    private let currencyFormatter = CurrencyFormatter(currencySettings: ServiceLocator.currencySettings)
```

Comment:
This reference is kept here because it's temporarily used for calculating the totals after checkout, at some point when that project moves forward this might be removed as well.